src/main/java/org/assertj/core/api/AbstractOffsetDateTimeAssert.java

```diff
@@ -671,6 +671,33 @@ public SELF isStrictlyBetween(String startExclusive, String endExclusive) {
     return isStrictlyBetween(parse(startExclusive), parse(endExclusive));
   }
 
+  /**
+   * Verifies that actual and given {@code OffsetDateTime} have the same {@link java.time.Instant}.
+   * <p>
+   * Example:
+   * <pre><code class='java'> // successful assertions
+   * OffsetDateTime OffsetDateTime1 = OffsetDateTime.of(2000, 12, 12, 3, 0, 0, 0, ZoneOffset.ofHours(3));
+   * OffsetDateTime OffsetDateTime2 = OffsetDateTime.of(2000, 12, 12, 0, 0, 0, 0, ZoneOffset.ofHours(0));
+   * assertThat(OffsetDateTime1).hasSameInstantAs(OffsetDateTime2);
+   *
+   * // failing assertions
+   * OffsetDateTime OffsetDateTime1 = OffsetDateTime.of(2000, 12, 12, 3, 0, 0, 0, ZoneOffset.ofHours(3));
+   * OffsetDateTime OffsetDateTime2 = OffsetDateTime.of(2000, 12, 12, 2, 0, 0, 0, ZoneOffset.ofHours(0));
+   * assertThat(OffsetDateTime1).hasSameInstantAs(OffsetDateTime2);</code></pre>
+   *
+   * @param other the given {@link java.time.OffsetDateTime}.
+   * @return this assertion object.
+   * @throws AssertionError if the actual {@code OffsetDateTime} is {@code null}.
+   * @throws IllegalArgumentException if other {@code OffsetDateTime} is {@code null}.
+   * @throws AssertionError if the actual {@code OffsetDateTime} does not have the same {@code Instant} as the other.
+   */
+  public SELF hasSameInstantAs(OffsetDateTime other) {
```

Comment:
After giving it more time, I think I got a better name: `isAtSameInstantAs`, the idea came after reading [OffsetDateTime.html#withOffsetSameInstant](https://docs.oracle.com/javase/8/docs/api/java/time/OffsetDateTime.html#withOffsetSameInstant-java.time.ZoneOffset-) which states:
> Returns a copy of this OffsetDateTime with the specified offset ensuring that the result **is at the same instant**.

Reply:
Got the method changed to `isAtSameInstantAs`.